server/static/js/main.js

```diff
@@ -19,3 +19,18 @@ if (typeof HTMLElement.prototype.removeClass !== "function") {
     }
 }
 
+$('body').on('click', 'button[data-confirm]', function(e) {
+  e.preventDefault();
+
+  var form = $(this).parents('form');
+  var confirmText = $(this).attr('data-confirm');
```

Comment:
`$(this).data('confirm')` might also work. I'm not actually sure which one is better. I think this is sufficient for our use case.